mmv1/products/compute/UrlMap.yaml

```diff
@@ -154,6 +154,21 @@ examples:
       http_health_check_name: 'health-check'
       storage_bucket_name: 'static-asset-bucket'
       error_backend_bucket_name: 'error-backend-bucket'
+  - name: 'url_map_http_filter_configs'
```

Comment:
Are these fields beta only? 

Could you add `min_version`: 'beta' to the tests you've added if it is the case?